lib/atc.js

```diff
@@ -0,0 +1,184 @@
+/**
```

Comment:
Why the rename to atc, does it have any specific meaning or is it just to fit the theme of flight related names?